docs/user_guide.md

```diff
@@ -37,6 +37,10 @@ GOOGLE_APPLICATION_CREDENTIALS=~/.config/gcloud/application_default_credentials.
 
 You can specify a [service account to impersonate](https://cloud.google.com/iam/docs/impersonating-service-accounts) for all Google API calls with the `GOOGLE_IMPERSONATE_SERVICE_ACCOUNT` environment variable.
 
+### Validating compliance IAM member & binding resources
```

Comment:
```suggestion
### Permission Requirements

The service account performing the validation must have the following permissions to avoid runtime issues fetching existing resources.

#### Security Reviewer role
```